tests/integration/test_ethereum_tester.py

```diff
@@ -190,6 +190,8 @@ def test_eth_getTransactionReceipt_unmined(self, eth_tester, web3, unlocked_acco
             eth_tester.enable_auto_mine_transactions()
             eth_tester.mine_block()
 
+    # TODO: replace auto mining switch with decorator
```

Comment:
👍

Reply:
ie~ would like to this as part of this PR, eventually :)